palindrome.js

```diff
@@ -0,0 +1,23 @@
+// Check if a word is a palindrome by a recursive function
+
+function checkPalindrome(word) {
+  let str = word.toLowerCase()
+  let wordStart = 0
+  let wordEnd = str.length - 1
+
+  // caso base: Se a palavra tem 1 ou 2 letras, é um palíndrome
+  if (str.length <= 1) {
```

Comment:
Referes que se tiver 1 ou 2 letras é palindrome mas no IF apenas válidas se tem 0 ou 1 letras. 
O correto será 
```suggestion
  if (str.length <= 2) {
```

Reply:
Tenta desta forma:

const word = “ABBA”
console.log(checkPalindrome(word, word))

Depois fazes console.log do primeiro parâmetro e slice do segundo parâmetro